sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
+These shared APIs allow client libraries to expose common functionality in a consistent fashion, 
```

Comment:
To play devil's advocate,  API is not defined within this document and is used as a well-known acronym.   Generally speaking, it's encouraged to define any acronym before using it to avoid assuming knowledge.

Reply:
In general I agree, but I think API when expanded is actually confusing.